docs/Deploying/Configuration.md

```diff
@@ -1184,14 +1184,35 @@ osquery:
   status_log_plugin: firehose
   result_log_plugin: firehose
 ```
+#### External Activity Audit Logging
 
-#### activity_enable_audit_log
+> Applies only to Fleet Premium. Acitivity information is available for all Fleet instances using the [Activities API](https://fleetdm.com/docs/using-fleet/rest-api#activities).
+
+Stream Fleet user activities to logs using Fleet's logging plugins.
+
+The following types of activity are included:
+
+Created pack
```

Comment:
You could link to https://github.com/fleetdm/fleet/blob/main/docs/Using-Fleet/Audit-Activities.md which is automatically generated from our golang code.